i18n/en.toml

```diff
@@ -8,6 +8,7 @@ other = "tag"
 other = "series"
 
 [reading_time]
+one = "One-minute read"
```

Comment:
I just found out that `1-minute` or `X-minute` translations are not wrong. So, it was not wrong to have just one translation for EN.
But, no problem. Adding `One-minute` is fine too.